engine/univSubst.ml

```diff
@@ -68,6 +68,10 @@ let subst_univs_fn_constr f c =
       let u' = fi u in
         if u' == u then t
         else (changed := true; mkConstructU (c, u'))
+    | Case (ci, u, pms, p, iv, c, br) ->
+      let u' = fi u in
+      if u' == u then map aux t
+      else (changed := true; map aux (mkCase (ci, u', pms, p, iv, c, br)))
     | _ -> map aux t
```

Comment:
I guess its absence means we used to do the wrong thing on caseinvert.

This function is only used for hintconstr in comhints AFAICT so I guess it just didn't come up.